sim_objects.py

```diff
@@ -0,0 +1,277 @@
+import math
+import numpy as np
+import pybullet as p
+from scipy.spatial.transform import Rotation as R
+
+PI = math.pi
+TWO_PI = 2 * math.pi
+
+# water bottle
+
+
+class Bottle:
+    WATER_DENSITY = 997    # kg/m³
+    VOL_TO_MASS = 0.0296   # fl-oz to kg
+    PLASTIC_MASS = 0.0127  # kg
+    PLANE_OFFSET = 0.056
+    INIT_PLANE_OFFSET = 0.03805010362200368  # found experimentally
+    # PLANE_OFFSET = 0
+    DEFAULT_RAD = 0.03175    # m
+    DEFAULT_HEIGHT = 0.1905  # m
+    DEFAULT_FRIC = 0.1  # plastic-wood dynamic friction
+    DEFAULT_FILL = 0.5
+
+    def __init__(self, start_pos, start_ori,
+                 fill_prop=DEFAULT_FILL,
+                 fric=DEFAULT_FRIC,
+                 radius=DEFAULT_RAD,
+                 height=DEFAULT_HEIGHT):
+        self.start_pos = start_pos
```

Comment:
By the way this applies to methods too, not just class variables.

Reply:
Ah yes, I am aware of this convention, but was just lazy about it. I will make this change later since it affects many more things, but it definitely would help me avoid making hacky fixes that directly overwrite the private variables, a sign that I should write a public function to do this.